plugins/woocommerce/i18n/locale-info.php

```diff
@@ -2896,38 +3093,41 @@
 		'currency_pos'   => 'left',
 		'thousand_sep'   => ',',
 		'decimal_sep'    => '.',
+		'negativity'     => '-',
 		'num_decimals'   => 2,
 		'weight_unit'    => 'kg',
 		'dimension_unit' => 'cm',
 		'direction'      => 'ltr',
 		'default_locale' => 'es_PR',
-		'name'           => 'United States (US) dollar',
+		'name'           => 'US Dollar',
 		'singular'       => 'US dollar',
 		'plural'         => 'US dollars',
 		'short_symbol'   => '$',
 		'locales'        => $locales['USD'],
 	),
 	'PS' => array(
-		'currency_code'  => 'JOD',
+		'currency_code'  => 'ILS',
 		'currency_pos'   => 'right_space',
 		'thousand_sep'   => '.',
 		'decimal_sep'    => ',',
-		'num_decimals'   => 3,
+		'negativity'     => '-',
+		'num_decimals'   => 2,
 		'weight_unit'    => 'kg',
 		'dimension_unit' => 'cm',
 		'direction'      => 'rtl',
 		'default_locale' => 'ar_PS',
-		'name'           => 'Jordanian dinar',
```

Comment:
See my comment on `plugins/woocommerce/tests/api-core-tests/tests/data/data-crud.test.js` -- even if this is just a normal non-political, technical fix for something, this seems like a thing we should make sure we all understand well before merging.

Reply:
Definitely I agree. I'm not sure how to convert this to accept multiple currencies for some countries, as some other countries use 2-3 currencies actively too. Thanks for bringing this up!

Reply:
I could confirm the currencies for: 

- Cuba: https://www.gov.uk/foreign-travel-advice/cuba/money
- Haiti: https://www.gov.uk/foreign-travel-advice/haiti/money
- Lesotho: https://www.gov.uk/foreign-travel-advice/lesotho/money

But Bhutan, Namibia, Panama and Palestine still is unknown.

Reply:
This probably warrants the same treatment as my comment on `plugins/woocommerce/tests/api-core-tests/tests/data/data-crud.test.js` -- let's trust and go with the Unicode recommendation here.